### ISSUE TITLE
Dropzone file param missing when using drag&drop

### ISSUE THREAD

**Issue by DaveEshopGuide** [baoagency/polaris_view_components#237]
When using the Dropzone component in a form and then submitting the form, the parameter for the file is not submitted to the server, **when Drag&Drop was used**. It works when picking the file from the dialog. 

Tried with and without multiple/accept options. 

```
<%= form_with( url:import_path, builder: Polaris::FormBuilder) do |form| %>
          <%= polaris_form_layout do |form_layout| %>
            <% form_layout.item do %>
              <%= form.polaris_dropzone :import_file,
                                        multiple: false,
                                        accept: 'text/csv',
                                        error_overlay_text: 'File type must be .csv'%>
            <% end %>
            <% form_layout.item do %>
              <%= polaris_button(primary: true, submit: true) { "Upload & Import" } %>
            <% end %>
          <% end %>
        <% end %>
```        

<img width="324" alt="image" src="https://user-images.githubusercontent.com/52373186/169838551-af10f3fe-d8ae-45d2-a0a0-e11ea6a3af38.png">

D&D:
<img width="516" alt="image" src="https://user-images.githubusercontent.com/52373186/169837183-dd21605a-cdcc-4080-b6d3-cdbc50ff610a.png">

File picked from dialog:
<img width="505" alt="image" src="https://user-images.githubusercontent.com/52373186/169837487-31f9cc32-b4f2-47b8-856a-211eea1b4c9f.png">

Using: 
ruby '3.1.2'
'rails', '~> 7.0.3'
polaris_view_components (0.10.0)

Windows 11 Machine, Chrome Browser

**Comment by kirillplatonov** [comment]
@DaveEshopGuide thanks for reporting that. I totally missed that bug. I opened a PR that should fix that: https://github.com/baoagency/polaris_view_components/pull/239

**Comment by kirillplatonov** [comment]
@DaveEshopGuide published [v0.10.1](https://github.com/baoagency/polaris_view_components/releases/tag/v0.10.1) with fix for that issue.
